Formula/corectl.rb

```diff
@@ -20,19 +20,22 @@ class Corectl < Formula
   def install
     ENV["GOPATH"] = buildpath
 
-    mkdir_p buildpath/"src/github.com/TheNewNormal/"
-    ln_s buildpath, buildpath/"src/github.com/TheNewNormal/#{name}"
+    path = buildpath/"src/github.com/TheNewNormal/#{name}"
+    path.install Dir["*"]
+
     Language::Go.stage_deps resources, buildpath/"src"
 
     args = []
     args << "VERSION=#{version}" if build.stable?
 
-    system "make", "corectl", *args
-    system "make", "documentation/man"
+    cd path do
+      system "make", "corectl", *args
+      system "make", "documentation/man"
```

Comment:
01:18am here as well, so fully understand any tiredness 😄. It's interesting that you got the extra warning there, I wonder why that popped up on CI but didn't for me here. Error was noticed by running a simple `brew install -s -v corectl` after pulling this PR locally.